resources/metadata/abapEnvironmentPullGitRepo.yaml

```diff
@@ -52,7 +52,19 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-        mandatory: true
+        mandatory: false
+      - name: repositories
+        type: string
+        description: Specifies a YAML file containing the repositories configuration
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: false
+        resourceRef:
```

Comment:
I removed the resourceRef from cloneGitRepo some days ago. IMO, this is not necessary anymore.

Reply:
`resourceRef` as well as `mandatory` parameter for all false values have been removed